contrib/config_option_database/Makefile.am

```diff
@@ -0,0 +1,25 @@
+CFG_OPT_DB_DIR = ${top_srcdir}/contrib/config_option_database
+
+EXTRA_DIST += \
+	$(CFG_OPT_DB_DIR)/config_database.py \
+	$(CFG_OPT_DB_DIR)/requirements.txt \
+	$(CFG_OPT_DB_DIR)/__init__.py \
+	$(CFG_OPT_DB_DIR)/utils/BisonGraph.py \
+	$(CFG_OPT_DB_DIR)/utils/ConfigOptions.py \
+	$(CFG_OPT_DB_DIR)/utils/MergeYm.py \
+	$(CFG_OPT_DB_DIR)/utils/OptionParser.py \
+	$(CFG_OPT_DB_DIR)/utils/Yacc2Graph.py \
+	$(CFG_OPT_DB_DIR)/utils/__init__.py \
+	$(CFG_OPT_DB_DIR)/utils/test/test_BisonGraph.py \
+	$(CFG_OPT_DB_DIR)/utils/test/test_OptionParser.py \
+	$(CFG_OPT_DB_DIR)/utils/test/test_Yacc2Graph.py
+
+if ENABLE_CONFIG_DATABASE
+contrib/config_option_database/.cache/options.json:
+	python3 -m pip install --user -r /requirements.txt
```

Comment:
We are detecting the python interpreter during configure.. You can use `$(PYTHON)`, similarly as in `lex-rules.am`.

Reply:
@furiel Is it means that $(PYTHON) variable name will work for python3 interpreter also?

Reply:
There is no `PYTHON3` variable now. The version `PYTHON` can be both 2 or 3, detected by some heuristics. The detection can be forced with the `--with-python=3` option.

The gain is the `PYTHON` variable will work on distributions where the binary is just called `python` for the version3 of the interpreter.